index.html

```diff
@@ -358,130 +358,130 @@ <h4 class="functionality__paragraph">Necesito Ayuda</h4>
+                  <span class="most-popular__star" aria-hidden="true">&#9733;</span>
+                  <span class="most-popular__rating" aria-label="calificación de productos 5">5</span>
                 </div>
               </span>
             </div>
-          </div>
-          <div class="most-popular__card most-popular__card--card-6 ">
-            <a href="#" class="most-popular__link"></a>
-            <p class="most-popular__badge most-popular__badge--red"> OFERTA</p>
-            <img class="most-popular__img" src="image/vacuum-cleaner.png" alt="">
+          </li>
+          <li class="most-popular__card most-popular__card--card-6 ">
+            <a href="#" class="most-popular__link" aria-label="ver Microonda 0.8 de acero inoxidable"></a>
+            <p class="most-popular__badge most-popular__badge--red" aria-hidden="true"> OFERTA</p>
+            <img class="most-popular__img" src="image/vacuum-cleaner.png" alt="Microonda 0.8 de acero inoxidable">
             <div class="most-popular__text">
               <span class="most-popular__name">Microonda 0.8 de acero inoxidable</span>
               <span class="most-popular__total">
-                <p class="most-popular__prices">$87.99 <s class="most-popular__number">$100.99</s>
+                <p class="most-popular__prices" aria-label="precio de producto $87.99">$87.99 <s class="most-popular__number" aria-label="precio anterior $100.99">$100.99</s>
                 </p>
                 <div class="most-popular__calification">
-                  <span class="most-popular__star">&#9733;</span>
-                  <span class="most-popular__rating">4</span>
+                  <span class="most-popular__star" aria-hidden="true">&#9733;</span>
+                  <span class="most-popular__rating" aria-label="calificación de productos 4">4</span>
                 </div>
               </span>
             </div>
-          </div>
-        </div>
+          </li>
+        </ul>
         <div class="most-popular__right">
-          <a href="#" class="most-popular__link"></a>
+          <a href="#" class="most-popular__link" aria-label="ver Desayunador con 2 bancas color espresso"></a>
           <div class="most-popular__right-method">
             <div class="most-popular__right-shipping">
-              <img class="most-popular__right-close" src="image/Close.png" alt="Close">
-              <p class="Close">Envío a domicilio</p>
+              <img class="most-popular__right-close" src="image/Close.png" alt="">
+              <p class="Close" aria-label="envio a domicilio">Envío a domicilio</p>
             </div>
             <div class="most-popular__right-removal">
-              <img class="most-popular__right-check" src="image/Check.png" alt="Check">
-              <p class="Check">Retiro en tienda</p>
+              <img class="most-popular__right-check" src="image/Check.png" alt="">
+              <p class="Check" aria-label="Retiro en tienda">Retiro en tienda</p>
```

Comment:
Aqui se lee mas como que esto fuera algun elemento interactivo para hacer el retiro en tienda
```suggestion
              <p class="Check" aria-label="Este producto esta disponible para retiro en tienda">Retiro en tienda</p>
```